packages/apollo-engine-reporting/src/extension.ts

```diff
@@ -61,7 +57,8 @@ export class EngineReportingExtension<TContext = any>
       options.generateClientInfo ||
       // Default to using the clientInfo field of the request's extensions, when
       // the ClientInfo fields are undefined, we send the empty string
-      (({ extensions }) => (extensions && extensions.clientInfo) || {});
+      (({ request }) =>
+        (request.extensions && request.extensions.clientInfo) || {});
```

Comment:
I wonder if we should start putting our extensions under an `apollo` key. We haven't done that so far with tracing and cache control, so maybe it's ok. But this seems even more product-specific.

Reply:
I don't think this is any more product specific than our other extensions fwiw -- it's just an accepted way to talk about client information.